CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+## 1.17.2
+
+* Accept Dart SDK versions above 3.0.
```

Comment:
but also drop SDK req to 2.17?

Reply:
That was a mistake. Don't know how. Fixed.